cluster/ubuntu/util.sh

```diff
@@ -369,6 +374,15 @@ function provision-master() {
     ubuntu/binaries/master/ \
     "${MASTER}:~/kube"
 
+  if [ -z "$CNI_PLUGIN_CONF" ] || [ -z "$CNI_PLUGIN_EXES" ]; then
+    # Flannel is being used: copy the flannel binaries and scripts, set reconf flag
+    scp -r $SSH_OPTS ubuntu/master-flannel/* "${MASTER}:~/kube"
+    NEED_RECONFIG_DOCKER=true
+  else
+    # CNI is being used: set reconf flag
```

Comment:
Can you also sanity check that the cni conf+bins exist? In fact, can you just grab the files from tarball at a url and install them instead (triggerd optionally, because you might be developing, or for the latter case you could supply a file:// url)?